matchbox_socket/src/integrations/bevy_plugin.rs

```diff
@@ -0,0 +1,33 @@
+use crate::{WebRtcSocket, WebRtcSocketBuilder};
+use bevy::{ecs::system::Command, prelude::*, tasks::IoTaskPool};
+
+/// A [`Resource`] wrapping [`WebRtcSocket`].
+///
+/// To create and destroy this resource use the [`OpenSocket`] and [`CloseSocket`] [`Command`]s respectively.
+///
+/// Optional to allow users to take ownership of the socket.
+#[derive(Debug, Resource)]
+pub struct MatchboxSocket(pub Option<WebRtcSocket>);
```

Comment:
I'm not sure I understood your idea outline in the other PR, but on top of #160, there's no need for this to be an `Option`.

If it's there, we're connected.

Reply:
Yeah, this is what I've just removed. It was this having to be an option that spurred me to create #160